activemodel/lib/active_model/errors.rb

```diff
@@ -231,17 +231,17 @@ def each(&block)
       else
         ActiveSupport::Deprecation.warn(<<-MSG.squish)
```

Comment:
Looks like this message expects new line as a structure, `squish` isn't matched in that case.
Can you change `squish` to squiggly heredoc?

```suggestion
        ActiveSupport::Deprecation.warn(<<~MSG)
```